sources/packages/web/src/components/common/OverawardDetails.vue

```diff
@@ -141,7 +147,7 @@ export default defineComponent({
   props: {
     studentId: {
       type: Number,
-      required: true,
+      required: false,
```

Comment:
I didn't understand why not required.

Reply:
studentId is optional when we use this component in the student page, as we fetch it from the student's UserToken